api/src/main/java/com/xwiki/googleapps/internal/CookieAuthenticationPersistenceImpl.java

```diff
@@ -75,39 +80,46 @@
     private static final String COOKIE_DOT_PFX = ".";
 
     private static final String EQUAL_SIGN = "=";
+
     private static final String UNDERSCORE = "_";
 
     @Inject
     private Logger logger;
 
-    private XWikiContext context;
+    @Inject
+    private Provider<XWikiContext> contextProvider;
 
     @Inject
-    private ComponentManager componentManager;
+    private GoogleAppsManager manager;
 
     private String cookiePfx;
```

Comment:
``cookiePrefix`` is way better and only 3 letters longer..

Reply:
Reformulated. Thanks.